package-lock.json

```diff
@@ -1570,7 +1570,8 @@
     "arr-flatten": {
       "version": "1.1.0",
       "resolved": "https://registry.npmjs.org/arr-flatten/-/arr-flatten-1.1.0.tgz",
-      "integrity": "sha512-L3hKV5R/p5o81R7O02IGnwpDmkp6E982XhtbuwSe3O4qOtMMMtodicASA1Cny2U+aCXcNpml+m4dPsvsJ3jatg=="
+      "integrity": "sha512-L3hKV5R/p5o81R7O02IGnwpDmkp6E982XhtbuwSe3O4qOtMMMtodicASA1Cny2U+aCXcNpml+m4dPsvsJ3jatg==",
```

Comment:
I didn't intentionally, I built and it changed

Reply:
Be sure you're using the correct node and npm versions (called out in the package.json/.nvmrc files).  Diffs like that typically occur because of differing node versions.  If you are on a different version, change to the correct one and re-install.

Reply:
I am using the correct versions, and did a new `npm install`. Came up with some other diffs.